.hlint.yaml

```diff
@@ -14,8 +14,10 @@
   - name: [ExistentialQuantification, MultiParamTypeClasses, NamedFieldPuns]
   - name: [FlexibleContexts, FlexibleInstances]
   - name: [PackageImports]
+  - {name: ConstraintKinds, within: [GHC.Util.Language.Haskell.GHC.ExactPrint.Types]}
   - {name: CPP, within: [HsColour]} # so it can be disabled to avoid GPL code
-  - {name: TypeFamilies, within: [GHC.Util.HsExpr, GHC.Util.FreeVars, Hint.Export, Hint.Restrict]} # type family constraints here
+  - {name: TypeFamilies, within: [GHC.Util.HsExpr, GHC.Util.FreeVars, Hint.Export, Hint.Restrict, Hint.Bracket]} # type family constraints here
+  - {name: RankNTypes, within: [GHC.Util.Language.Haskell.GHC.ExactPrint.Types]}
```

Comment:
Again, RankNTypes and TypeFamilies are inoffensive.

Reply:
✔️